jupyterhub_singleuser_profiles/ui/src/ImageForm/ImageSelector.tsx

```diff
@@ -33,7 +33,7 @@ const ImageSelector: React.FC<ImageSelectorProps> = ({
     return <ImageTagPopover tag={currentTag} description={image.description || undefined} />;
   };
 
-  const disabled = image.tags.every((tag) => !isImageTagBuildValid(tag));
+  const disabled = image.tags?.every((tag) => !isImageTagBuildValid(tag)) ?? true;
```

Comment:
question/change:
if `!isImageTagBuildValid(...)` is returning a bool will the nullish operator fire? Testing `false ?? true` returns `false`

Reply:
Upated, but the nullish operator only fires on `undefined` or `null`, false would not trigger it.